core/tauri-build/src/mobile.rs

```diff
@@ -152,8 +137,6 @@ pub fn inject_android_project(
 }
 
 fn copy_folder(source: &Path, target: &Path, ignore_paths: &[&str]) -> Result<()> {
-  let _ = remove_dir_all(target);
```

Comment:
`android dev --release` didn't work unless I remove this, because `build` folder was being used while tauri's build script was running and it crashed every time

Reply:
doesn't make much sense, it should also happen without the --release flag. I just tried it and it works fine. Might be some other issue.

Reply:
maybe something else on my system locked the directory, lets move on without it then